src/mobu/dependencies/manager.py

```diff
@@ -23,7 +18,7 @@ class MonkeyBusinessManager:
     """Manages all of the running monkeys."""
 
     def __init__(self) -> None:
-        self._monkeys: Dict[str, Monkey] = {}
+        self._flocks: Dict[str, Flock] = {}
```

Comment:
So the base unit of MonkeyBusiness is no longer a single monkey, but instead a flock?

Reply:
Right, which can be a single monkey.